galaxy/templates/deployment-job.yaml

```diff
@@ -137,7 +137,7 @@ spec:
               subPath: probedb.py
             {{- range $key, $entry := $.Values.extraFileMappings -}}
             {{- if $entry.applyToJob }}
-            - name: extra-files-{{ include "galaxy.getFilenameFromPath" $key | replace "." "-" }}
+            - name: extra-files-{{ include "galaxy.getFilenameFromPath" $key | replace "." "-" | replace "_" "-" }}
```

Comment:
I think this should be moved to the helper template itself, or have some kind of utility function like `make_k8s_label_compatible` or something?

Reply:
Yes! i'll just call it `make_dns_compliant` for now

Reply:
This is all done now, and handled more general cases and the corner case where two files have the same filename but different paths.